<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="StationList_Table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ScanLocalStations_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Scan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearList_Button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear table</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseWindow_Button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>410</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExportCSV_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Export CSV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FreqMonitor_Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="WaitTime_Edit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="WaitTime_Label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wait time [s]</string>
   </property>
  </widget>
  <widget class="QLabel" name="RDSTimeout_Label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RDS timeout [s]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RDSTimeout_Edit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>340</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ImportLocalStations_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>20</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="LocalStations_Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Local stations</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LocalStations_Label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DX Frequency Monitor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LocalStation_Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>No. of local stations:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LocalStationNo_Label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="RDSTimeout_Label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Upper freq. limit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UpperFreqLimit_Edit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
